src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -42,6 +44,15 @@ public static string GetOperationTree(IOperation operation, int initialIndent =
             return walker._builder.ToString();
         }
 
+        public static void Verify(string expectedOperationTree, string actualOperationTree)
+        {
+            char[] newLineChars = Environment.NewLine.ToCharArray();
+            string actual = actualOperationTree.Trim(newLineChars);
+            expectedOperationTree = expectedOperationTree.Trim(newLineChars);
+            expectedOperationTree = Regex.Replace(expectedOperationTree, "([^\r])\n", "$1" + Environment.NewLine);
```

Comment:
hmm.. but you trimmed new lines right above, regex so, there will be no \r or \n in text anymore?